buffer/buffer.go

```diff
@@ -747,21 +786,34 @@ func (buffer *Buffer) NewLine() {
 	}
 }
 
+func (buffer *Buffer) SetNewLineMode() {
+	buffer.lineFeedMode = false
+}
+
+func (buffer *Buffer) SetLineFeedMode() {
+	buffer.lineFeedMode = true
+}
+
+func (buffer *Buffer) NewLineMode() bool {
```

Comment:
will it fit better to name the method `IsNewLineMode()`?

Reply:
This was what I originally wrote but it won't fit with AutoWrap() identifier, will it?

Reply:
@MaxRis  oh, I haven't committed that yet.
I got the idea. The private member is autoWrap, the public is IsAutoWrap(), The private is newLineMode, the public is IsnewLineMode(), right?

Reply:
This should work! Thanks!